terraform/app/modules/ecs_service/main.tf

```
@@ -0,0 +1,100 @@
+resource "aws_security_group" "this" {
+  name        = "${var.server_type}-service-${var.environment}"
+  description = "Security Group for communication with Background ECS Service"
+  vpc_id      = var.network_params.vpc_id
+  lifecycle {
+    ignore_changes = [description]
+  }
+}
+
+resource "aws_security_group_rule" "egress_all" {
+  type              = "egress"
+  from_port         = 0
+  to_port           = 0
+  protocol          = "-1"
+  cidr_blocks       = ["0.0.0.0/0"]
+  security_group_id = aws_security_group.this.id
+}
+
+resource "aws_ecs_service" "this" {
+  name                              = "mavis-${var.environment}-${var.server_type}"
+  cluster                           = var.cluster_id
+  task_definition                   = aws_ecs_task_definition.this.arn
+  desired_count                     = var.desired_count
+  launch_type                       = "FARGATE"
+  enable_execute_command            = true
+  health_check_grace_period_seconds = 60
+
+  network_configuration {
+    subnets         = var.network_params.subnets
+    security_groups = [aws_security_group.this.id]
+  }
+  deployment_controller {
+    type = var.deployment_controller
+  }
+  dynamic "deployment_circuit_breaker" {
+    for_each = var.deployment_controller == "ECS" ? [1] : []
+    content {
+      enable   = true
+      rollback = true
+    }
+  }
+  dynamic "load_balancer" {
+    for_each = var.loadbalancer != null ? [1] : []
+    content {
+      target_group_arn = var.loadbalancer.target_group_arn
+      container_name   = "mavis-application"
```

```suggestion
      container_name   = "application"
```

Do we need the prefix?

---

Definitely not needed, the question is just where we add the mavis prefix and were we dont

---

I would suggest avoid a `mavis-` prefix unless we really need one, I think it's obvious that all of this relates to Mavis.